guides/release/applications/dependency-injection.md

```diff
@@ -234,16 +234,18 @@ For example, this component plays songs with different audio services based
 on a song's `audioType`.
 
 ```javascript {data-filename=app/components/play-audio.js}
-import Component from '@ember/component';
-import { computed } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
 import { getOwner } from '@ember/application';
 
 // Usage:
 //
-// {{play-audio song=song}}
+// <PlayAudio @song=this.song />
 //
-export default Component.extend({
-  audioService: computed('song.audioType', function() {
+export default class PlayAudioComponent extends Component {
+  @tracked song;
```

Comment:
I wouldn't use `@tracked` at all here (once migrated to `@glimmer/component`), I'd probably do:

```js
export default class PlayAudioComponent extends Component {
  get audioService() {
    if (!this.args.song) {
      return null;
    }

    let applicationInstance = getOwner(this);
    let { audioType } = this.args.song;

    return applicationInstance.lookup(`service:audio-${audioType}`);
  }

  click() {
    let player = this.audioService;
    player.play(this.args.song.file);
  }
}
```